uwblueprint/don-efficace
# Create View for Donation Form

## Notion ticket link
[Create View for Donation Form](https://www.notion.so/uwblueprintexecs/Create-View-for-Donation-Form-10710f3fb1dc800eb61fd6934ab25155?pvs=4)



## Implementation description
* Created a DonationForm component with a hero banner, section titles, and multi-select checkboxes using Chakra UI components
* Created a route to view the Donation Form page at /donate



## Steps to test
1. Navigate to /donate and ensure the page details match the Figma design
2. Ensure that the multi-select checkboxes check and uncheck as expected



## What should reviewers focus on?
* How closely the page resembles the Figma design


## Checklist
- [✅] My PR name is descriptive and in imperative tense
- [✅] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [✅] I have run the appropriate linter(s)
- [✅] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR
